src/Datadog.NativeLoader/logging.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include <iostream>
+#include <mutex>
+
+#include "string.h"
+
+template <typename Arg>
+std::string LogToString(Arg const& arg)
+{
+    return ToString(arg);
+}
+
+template <typename... Args>
+std::string LogToString(Args const&... args)
+{
+    std::ostringstream oss;
+    int a[] = {0, ((void) (oss << LogToString(args)), 0)...};
+    return oss.str();
+}
+
+template <typename... Args>
+void Verbose(const Args... args)
```

Comment:
Yes I've added this one because I was using it to write the Callback events to know the order and the event mask working. But writing those can be massive. So I leave it as a verbose mode, and switch to a normal Debug for everything else. This can be remove it along with the callers if you wish.